stdpopsim/dfe.py

```diff
@@ -22,7 +43,7 @@ class MutationType(object):
     convert_to_substitution = attr.ib(default=True, type=bool)
 
     def __attrs_post_init__(self):
-        if self.dominance_coeff < 0 or not np.isfinite(self.dominance_coeff):
```

Comment:
whoops - turns out that dominance coefficients *can* be less than zero (that's heterozygote disadvantage, or underdominance)